{
  "description": "An issue was discovered in the Linux kernel before 5.0. The function __mdiobus_register() in drivers/net/phy/mdio_bus.c calls put_device(), which will trigger a fixed_mdio_bus_init use-after-free. This will cause a denial of service.",
  "releases": {
    "bullseye": {
      "fixed_version": "4.19.37-1",
      "repositories": {
        "bullseye": "5.6.14-2"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "buster": {
      "fixed_version": "4.19.37-1",
      "repositories": {
        "buster": "4.19.118-2",
        "buster-security": "4.19.118-2+deb10u1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "jessie": {
      "fixed_version": "3.16.68-1",
      "repositories": {
        "jessie": "3.16.56-1+deb8u1",
        "jessie-security": "3.16.84-1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "sid": {
      "fixed_version": "4.19.37-1",
      "repositories": {
        "sid": "5.6.14-2"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "stretch": {
      "fixed_version": "4.9.168-1",
      "repositories": {
        "stretch": "4.9.210-1",
        "stretch-security": "4.9.210-1+deb9u1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    }
  },
  "scope": "local"
}